src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeNamesFormatter.java

```diff
@@ -43,20 +43,23 @@ public String getKey() {
     }
 
     @Override
-    public String format(String value) {
-        Objects.requireNonNull(value);
+    public String format(String nameList) {
+        Objects.requireNonNull(nameList);
         // Handle case names in order lastname, firstname and separated by ","
         // E.g., Ali Babar, M., Dingsøyr, T., Lago, P., van der Vliet, H.
-        if (!value.contains(" and ") && !value.contains("{") && !value.contains(";")) {
-            String[] valueParts = value.split(",");
+        if (!nameList.contains(" and ") && !nameList.contains("{") && !nameList.contains(";")) {
```

Comment:
I recently ran into the same issue (after a user noted it) in the office xml export, make sure to check for and case independent : `nameList.toUpperCase(Locale.ENGLISH).contains(" AND "))`

And I would suggest adding a test for it, to make sure the behaviour stays the same in future ;)